source/_integrations/spotify.markdown

```diff
@@ -12,25 +12,25 @@ ha_codeowners:
 ha_domain: spotify
 ---
 
-The `spotify` media player integration allows you to control [Spotify](https://www.spotify.com/) playback from Home Assistant.
+The `Spotify` media player integration allows you to control [Spotify](https://www.spotify.com/) playback from Home Assistant.
 
 ## Prerequisites
 
-- Spotify account
-- Spotify application, properly configured (see below).
+- `Spotify` account
+- `Spotify` application configured for Home Assistant (see [below](#create-a-spotify-application))
 
 <div class='note'>
-  Controlling the Spotify integration (pause, play, next, etc.) requires a Premium account.
-  If you do not have a Premium account, the integration in the frontend will not show the controls.
+  Spotify integrated media controls (play, pause, skip) requires a Premium account.
+  If you do not have a Premium account, the integration's frontend will not show the controls.
 </div>
 
-To create the required Spotify application:
+### Create a Spotify application
 
 - Login to [Spotify Developer](https://developer.spotify.com).
 - Visit the [My Applications](https://developer.spotify.com/my-applications/#!/applications) page.
 - Select **Create An App**. Enter any name and description.
-- Once your application is created, view it and copy your **Client ID** and **Client Secret**, which are used in the Home Assistant configuration file.
-- On the Spotify page, enter the **Edit Settings** dialog of your newly created application and add a **Redirect URI**:
+- Once your application is created, view it and copy your `Client ID` and `Client Secret`, which are used in the Home Assistant configuration file.
```

Comment:
Please restore, Client ID & Secret to not refer to code and should not be code fenced.

Reply:
Sorry. I am new to this. I'm learning what should and should not be code fenced